index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Erick's Portfolio</title>
+</head>
+ <body>
+     <header>
+      <nav id="nav-bar">
+            <a href="#">
+            <div class="bar"></div>
+            <div class="bar"></div>
+            <div class="bar"></div>
+            </a>
+      </nav>
+    </header>
+     <main>
+        <section class="content">
+            <div><h1 id="head1">Hey there.<br> I’m Erick.</h1>
+               <h2 id="head2">I’m a software <br> developer</h2>
+            </div>
+           <p class="intel">
+            I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.
+          </p>
+           <ul class="icon-box">
+            <li><a href="#"><img src="./features/icons/twitter.svg" alt="twitter icon"></a></li>
+            <li><a href="#"><img src="./features/icons/Linkedin icon.svg" alt="linked in"></a></li>
+            <li><a href="#"><img src="./features/icons/medium.svg" alt="medium icon"></a></li>
+            <li><a href="#"><img src="./features/icons/github.svg" alt="github icon"></a></li>
+            <li><a href="#"><img src="./features/icons/angellist.svg" alt="angellist"></a></li>
+           </ul>
+           <img id="img" src="/features/images/Disabled.png">
```

Comment:
- [ ] Kindly make sure you use the correct path for your images as you may notice how this image is not showing.

![image](https://github.com/Bupilipili/MY-PORTFOLIO/assets/97100091/664a5fe6-a411-420b-83ea-2949fbdb843f)
Hint: You may consider adding the dot to specify the root: `<img id="img" src="./features/images/Disabled.png">`